src/components/templates/Playa/AvatarPartygoers.tsx

```diff
@@ -15,7 +15,8 @@ const AvatarPartygoers: React.FC<PropsType> = ({
   state,
   useProfilePicture,
 }) => {
-  const { recentVenueUsers } = useRecentVenueUsers();
+  // @debt we don't have access to venueName here (but is legacy Playa template code to be removed so we don't mind), this is basically a noop
+  const { recentVenueUsers } = useRecentVenueUsers({ venueName: undefined });
```

Comment:
Mostly because I used the `ReactHook` helper type and wanted to write proper input/output types for it. And I think also because I wanted to make it super explicit that it was a `venueName`, not a `venueId`, and to have that obvious when it's refactored as well, as per this `@debt` comment:

![image](https://user-images.githubusercontent.com/753891/124706456-a12b5180-df3a-11eb-8537-4eeb60d8d992.png)